src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/UpdateOrgNameSraIdIntegrationTest.java

```diff
@@ -0,0 +1,89 @@
+package uk.gov.hmcts.reform.professionalapi;
+
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.professionalapi.controller.request.OrganisationCreationRequest;
+import uk.gov.hmcts.reform.professionalapi.util.AuthorizationEnabledIntegrationTest;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.professionalapi.helper.OrganisationFixtures.organisationRequestWithAllFields;
+import static uk.gov.hmcts.reform.professionalapi.helper.OrganisationFixtures.organisationRequestWithAllFieldsAreUpdated;
+
+
+class UpdateOrgNameSraIdIntegrationTest extends AuthorizationEnabledIntegrationTest {
+
+
+    @Test
+    void update_name_or_sra_of_an_active_organisation_with_prd_admin_role_should_return_200() {
+        Map<String, Object> updateResponse = professionalReferenceDataClient
+                .updateOrgNameSraIdStatus(organisationRequestWithAllFields().build(),  hmctsAdmin,getOrganisationId());
+
+        assertThat(updateResponse).containsEntry("http_status", 200);
```

Comment:
either of the query param should be present in the request, if both are null or empty then return 400 with error message "Name or SRA ID is required"

Reply:
added update_name_or_sra_with_invalid_name_and_sra_should_return_400 fro the same